frontend/packages/console-shared/src/types/backend-api.ts

```diff
@@ -0,0 +1,9 @@
+export type DevConsoleEndpointRequest = {
+  allowAuthHeader?: boolean;
```

Comment:
Do certain endpoints always require auth and others don't? I'm just wondering if we need this in the request. It would be cleaner to allow the backend to decide whether to send the auth header only when it's needed instead of having to trust what the client says.